framework/encode/vulkan_capture_manager.cpp

```diff
@@ -2779,14 +2779,18 @@ void VulkanCaptureManager::PostProcess_vkCreateGraphicsPipelines(VkResult
                 vulkan_wrappers::GetWrapper<vulkan_wrappers::PipelineWrapper>(pPipelines[p]);
             assert(ppl_wrapper != nullptr);
 
-            for (uint32_t s = 0; s < pCreateInfos[p].stageCount; ++s)
+            const auto binary_info = graphics::vulkan_struct_get_pnext<VkPipelineBinaryInfoKHR>(&pCreateInfos[p]);
+            if (!binary_info || binary_info->binaryCount == 0)
```

Comment:
nit picking: According to our coding guidelines the checks should be the other way around:
`if (binary_info == nullptr || !binary_info->binaryCount)`
Same applies to the other 2 functions which have the same if condition